examples/hata_basic_queue/bot.py

```diff
@@ -258,15 +250,19 @@ async def user_voice_update(client, event, _old):
         )
 
     logging.error(
-        "The `voice_server_update` event is not exposed by HATA, so the only way to use lavasnek_rs with it RN is to use the lavasnek discord gateway to connect."
+        (
+            "The `voice_server_update` event is not exposed by HATA,"
+            " so the only way to use lavasnek_rs with it RN is to use the lavasnek discord gateway to connect."
+        )
     )
     sys.exit(0)
 
-    @bot.events
-    async def voice_server_update(client, event):
-        await client.data.lavalink.raw_handle_event_voice_server_update(
-            event.guild_id, event.endpoint, event.token
-        )
+    # This is commented out because the `voice_server_update` event is not exposed by HATA,
+    # so the only way to use lavasnek_rs with it RN is to use the lavasnek discord gateway to connect.
```

Comment:
Do you really need to make the comment, having the doc avobe?

Reply:
I know that I didn't read it and was confused. But if you want it gone i understand

Reply:
Well, it effectively does nothing, just pointing it out.
Commenting the code is good, because you probably get a warning from that `sys.exit(0)`

Reply:
But do you want that comment removed?

Reply:
You choose, but resolve this conversation, so this can be merged.

Reply:
I'd say leave it in